studio/components/interfaces/SQLEditor/UtilityPanel/ResultsDropdown.tsx

```diff
@@ -110,7 +110,9 @@ const ResultsDropdown = ({ id }: ResultsDropdownProps) => {
       }
     >
       <Button asChild type="text" iconRight={<IconChevronDown />}>
-        <span>Results</span>
+        <span>
+          Results {result && result.rows.length > 0 && `(${result.rows.length.toLocaleString()})`}
```

Comment:
Small nit: the previous count still shows while the next query is running – maybe you can add another check here to make sure the results are displaying?